tests/api/admin/controller/test_work_editor.py

```diff
@@ -1007,6 +1032,11 @@ def test_process_cover_image(self):
         )
         assert root_mean_square < 10
 
+        # Remove temporary files if we've gotten this far. Assertion failures should leave
+        # the files intact for manual inspection.
+        for f in [tmpfile_before_0, tmpfile_before_1, tmpfile_after_0, tmpfile_after_1]:
```

Comment:
A failing test case would fail cleanup, maybe have this in a teardown function or in exception handling

Reply:
That was intentional; if the test fails, we want those images left in the filesystem so that they can be inspected visually.